implementations/rust/ockam/ockam_ffi/include/vault.h

```diff
@@ -113,7 +113,7 @@ ockam_vault_extern_error_t ockam_vault_secret_export(ockam_vault_t        vault,
                                                      ockam_vault_secret_t secret,
                                                      uint8_t*             output_buffer,
                                                      uint32_t             output_buffer_size,
-                                                     size_t*              output_buffer_length);
+                                                     uint32_t&            output_buffer_length);
```

Comment:
Hmm, this should be `uint32_t*`, not `uint32_t&` (`uint32_t&` is a C++-only type of pointer).

Reply:
Thanks for bringing that to my attention. dealing with FFI can really get my wires crossed.